cdc-accelerator/core/sitesCache.js

```diff
@@ -40,4 +38,27 @@ export default class SitesCache {
     static getSiteInfo(_apiKey) {
         return SitesCache.cache.find(({ apiKey }) => apiKey === _apiKey)
     }
+
+    static isCacheUpdated(configuration) {
+        if (!configuration?.cache) {
+            return false
+        }
+        let updated = true
+        if (configuration.source) {
+            configuration.source.forEach((entry) => {
+                updated &= configuration.cache.some(({ apiKey }) => {
+                    return entry.apiKey === apiKey
+                })
+            })
+        }
+
+        if (updated && configuration.deploy) {
+            configuration.deploy.forEach((entry) => {
+                updated &= configuration.cache.some(({ apiKey }) => {
+                    return entry.apiKey === apiKey
+                })
+            })
+        }
+        return updated ? true : false
```

Comment:
If `updated` stays a boolean, there is no need for this expression.

Reply:
changed